### ISSUE TITLE
Can I use the pretrained BERT-Base model directly for predict isNextSentence task?

### ISSUE THREAD

**Issue by nomadlx** [huggingface/transformers#5797]
# ❓ Can I use the pretrained BERT-Base model directly for predict isNextSentence task?

## Details

I has a document-level corpus, but it don't have document boundaries.
I want to confirm that whether can I predict isNextSentence task by pretrained BERT-Base model? The model does not use any data finetune and don't mask any token when i used it. 
Is prediction reliable by this way?

**Comment by nomadlx** [comment]
I have find anwser in [code](https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_bert.py#L1138).
